source/MRViewer/MRRenderPointsObject.cpp

```diff
@@ -376,7 +377,7 @@ RenderBufferRef<VertId> RenderPointsObject::loadValidIndicesBuffer_()
             if ( validPoints.test( v ) )
                 buffer[v / step] = VertId( v / step );
             else
-                buffer[v / step] = firstValid;
+                buffer[v / step] = VertId( firstValid / step );
```

Comment:
`firstValid / step` can be invalid
![image](https://github.com/MeshInspector/MeshLib/assets/120382683/0cc2ec9a-d984-4777-a0ed-3b0ea6226c38)
[extra_point2.zip](https://github.com/MeshInspector/MeshLib/files/15468442/extra_point2.zip)

Reply:
@rodion-orlov 
> E.g. if ( firstValid.valid() ) is always true, could be changed to assert.

It is not true if validPoints.none()

Reply:
> @rodion-orlov
> 
> > E.g. if ( firstValid.valid() ) is always true, could be changed to assert.
> 
> It is not true if validPoints.none()
There's a check above:
``
if ( !( dirty_ & DIRTY_POSITION ) || !objPoints_->hasVisualRepresentation() )
  return glBuffer.prepareBuffer<VertId>( validIndicesSize_, !validIndicesBuffer_.valid() );
``
``
bool ObjectPointsHolder::hasVisualRepresentation() const
{
    return points_ && points_->validPoints.any();
}
``
It is not clear what would be supposed to happen if the check failed.